desktop-themes/HighContrastInverse/metacity-1/metacity-theme-1.xml

```diff
@@ -147,7 +147,7 @@
 </draw_ops>
 
 <draw_ops name="outer_bevel">
-  <rectangle color="gtk:fg[NORMAL]"
+  <rectangle color="gtk:fg[NORMAL]" filled="true"
```

Comment:
Better 
*<rectangle color="gtk:bg[NORMAL]" filled="true"*
Otherwise menu/min/max/close buttons are invisible.

Reply:
hmm, using bg color here makes the white window border invisible.

Reply:
ah, indeed I had changed that in my test environment but that was missed in the patch.

How is it making the borders invisible? I do get white-on-black borders.